bennu-io/src/main/java/org/fenixedu/bennu/io/domain/GenericFile.java

```diff
@@ -258,13 +238,11 @@ public InputStream getStream() {
     public static void convertFileStorages(final FileStorage fileStorageToUpdate) {
         if (fileStorageToUpdate != null) {
             try {
-                for (final GenericFile genericFile : FileSupport.getInstance().getFileSet()) {
-                    if (fileStorageToUpdate == genericFile.getFileStorage() && fileStorageToUpdate != genericFile.getStorage()) {
-                        genericFile.updateFileStorage();
-                    }
-                }
+                FileSupport.getInstance().getFileSet().stream()
+                        .filter(gf -> fileStorageToUpdate == gf.getFileStorage() && fileStorageToUpdate != gf.getStorage())
+                        .forEach(gf -> gf.updateFileStorage());
```

Comment:
this can be changed to use the method reference `GenericFile::updateFileStorage`